docs/source/markdown/podman-build.1.md

```diff
@@ -272,6 +272,16 @@ BUILDAH\_ISOLATION environment variable.  `export BUILDAH_ISOLATION=oci`
 
 Add an image *label* (e.g. label=*value*) to the image metadata. Can be used multiple times.
 
+Users can set a special label **io.containers.capabilities=CAP1,CAP2,CAP3** that
+indicates the default list of Linux capabilities required for the container to
+run properly. Setting this label in a container or a container image tells podman
+to run the container with just these capabilties. Podman will launch the container
+with just the specified capabilties, as long as this list of capabilities is a
+subset of the default list.
+
+If the user or image specified capabilities that are not in the default set, the container
```

Comment:
I find "user or image" to be a bit confusing.  Could we just drop that?  "If the specified capabilities..."

Reply:
ooops, if dropped, the following "that" doesn't work.   "If the specified capabilities are not in the default set, ..."

Reply:
Fixed